charts/vineyard-operator/README.md

```diff
@@ -29,6 +29,23 @@ helm uninstall vineyard-operator
 
+## Values
+
+The following table lists the configurable parameters of the Vineyard Operator chart and their default values.
+
+| Key                | Type   | Default                                                                   | Description                                                                                                                |
+| ------------------ | ------ | ------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| replicaCount       | int    | 1                                                                         | The replica of vineyard operator.                                                                                          |
+| image.repository   | string | "vineyardcloudnative/vineyard-operator"                                   | The repository of vineyard operator image.                                                                                 |
+| image.pullPolicy   | string | "IfNotPresent"                                                            | The pull policy of vineyard operator image.                                                                                |
+| image.tag          | string | "latest"                                                                  | The image tag of vineyard operator image.                                                                                  |
+| serviceAccountName | string | "Vineyard-manager"                                                        | The service account name of vineyard operator.                                                                             |
+| service.type       | string | "ClusterIP"                                                               | The type of the service.                                                                                                   |
+| service.port       | int    | 9600                                                                      | The internal port of vineyard operator service                                                                             |
+| resources          | object | {limits: {cpu: 500m, memory:500Mi}},{requests: {cpu: 500m, memory:500Mi}} | The limits and requests of vineyard operator.                                                                              |
+| tolerations        | object | {}                                                                        | Tolerations allow the scheduler to schedule pods with matching taints                                                      |
+| affinity           | object | {}                                                                        | Affinity enables the scheduler to place a pod either on a group of nodes or a pod relative to the placement of other pods. |
+
```

Comment:
Could you please make this table less wider as well?

Reply:
Get it.